Eatery Blue/Eatery Blue/Analytics/Analytics.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  Analytics.swift
+//  AppDevAnalytics
+//
+//  Created by Kevin Chan on 9/4/19.
+//  Copyright © 2019 Kevin Chan. All rights reserved.
+//
+
+import FirebaseAnalytics
+
+protocol Payload {
```

Comment:
Could also use an enum instead of protocol here, but up to you

Reply:
I was just trying to model this off old eatery analytics to keep consistency